docs/generate_extension_db.py

```diff
@@ -33,6 +33,24 @@ def IsMissing(value):
   return value == '(missing)'
 
 
+def NumReadFiltersFuzzed():
+  f = open('test/extensions/filters/network/common/fuzz/uber_per_readfilter.cc', 'r')
```

Comment:
Suggest using `pathlib` and `read_text` here.

Reply:
I was hesitant to load the entire file because I only want the first `n` lines, so I preferred `n` `readline()`s. Hm... I will use `pathlib` though.

Reply:
Eh, if it's not a concern to load the entire file, oh well.

Reply:
The Pythonic way of doing things generally is to write for clarity/conciseness and then worry about performance when it really matters. It's one way of doing things.